src/shogun/lib/SGMatrix.h

```diff
@@ -160,11 +163,17 @@ template<class T> class SGMatrix : public SGReferencedData
 		 */
 		SGVector<T> get_column(index_t col) const;
 
-		/** Copy the content of a vector into a column
+		/** Copy the content of a vector lvalue reference into a column
```

Comment:
not sure if it makes sense to says this explicitly in the docs? Docs should probably not mention these things

Reply:
I think it does
Is not user facing anyways